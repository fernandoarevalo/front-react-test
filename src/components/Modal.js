import React from "react";
// Utils
import { MODAL } from "utils/constants/Contants";

export default function Modal() {
  return (
    <div>
      <div className="modal-content">
        <h4>{MODAL.TITLE}</h4>
        <p>{MODAL.MESSAGE}</p>
      </div>
    </div>
  );
}
