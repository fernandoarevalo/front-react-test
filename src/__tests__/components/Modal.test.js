import React from "react";
import { render, cleanup, fireEvent, screen } from "@testing-library/react";
import Modal from "components/Modal";

describe("<Modal />", () => {
  afterEach(() => {
    cleanup();
  });

  test("Snapshot Modal component", () => {
    const { asFragment } = render(<Modal />);
    expect(asFragment()).toMatchSnapshot();
  });
});
