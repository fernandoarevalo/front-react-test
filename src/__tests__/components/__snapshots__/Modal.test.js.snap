// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Modal /> Snapshot Modal component 1`] = `
<DocumentFragment>
  <div>
    <div
      class="modal-content"
    >
      <h4>
        Grandioso!
      </h4>
      <p>
        Tu información fue enviada con éxito, estaremos en contacto contigo
      </p>
    </div>
  </div>
</DocumentFragment>
`;
